chrome/content/zotero/xpcom/data/item.js

```diff
@@ -4181,6 +4181,7 @@ Zotero.Item.prototype.toJSON = Zotero.Promise.coroutine(function* (options) {
 	
 	obj.dateAdded = Zotero.Date.sqlToISO8601(this.dateAdded);
 	obj.dateModified = Zotero.Date.sqlToISO8601(this.dateModified);
+	if(obj.accessDate) obj.accessDate = Zotero.Date.dateToISO(Zotero.Date.sqlToDate(obj.accessDate));
```

Comment:
I think that `dateAdded` and `dateModified` are stored in UTC, so `toJSON` maintains that here. Whether we want `accessDate` exported in UTC, I'm not entirely sure. IIRC, it's stored and treated as local date within the client, but for API JSON format (being an interchange format), it would make sense to export in UTC (idk if that's what happens now for web API).